src/main/java/me/totalfreedom/totalfreedommod/command/Command_manageshop.java

```diff
@@ -11,7 +11,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.SENIOR_ADMIN, source = SourceType.ONLY_CONSOLE)
-@CommandParameters(description = "Manage the shop", usage = "/<command> <coins: <add | set | remove> <amount> <player | all> | items: <give | take> <item> <player>", aliases = "ms")
+@CommandParameters(description = "Manage the shop.", usage = "/<command> <coins: <add | set | remove> <amount> <player | all> | items: <give | take> <item> <player>", aliases = "ms")

```

Comment:
With this entire file, do coin values contain a decimal place? I'm not sure I'd want to be potentially confusing currency and punctuation in this one?

Reply:
No, coin values never should contain a decimal place, or at least, they aren't supposed to, i have no clue what happens if you try to /ms coins give <player> 0.1